website/docs/feature-flag-tutorials/use-cases/security-compliance.md

```diff
@@ -0,0 +1,239 @@
+-   Sync group membership in real-time.
+-   Ensure consistent access across multiple platforms.
+
+To unlock these benefits, set up [SCIM for automatic provisioning using our how-to guides](/how-to/provisioning).
+
+## Configure role-based access control for administrators and developers
+
+Now that you understand how Unleash handles authentication securely, let’s explore what needs to happen once your teams have access to the platform. When enterprise auditors evaluate software tools and development practices, they focus intensely on access control mechanisms.
+
+Consider this scenario: a Fortune 500 company is undergoing its annual security audit, and the auditors are specifically examining how feature flags are managed across their software development lifecycle. Without role-based access controls (RBAC), this company would face significant compliance risks and potential security vulnerabilities.
+
+[Role-based access control](/reference/rbac) allows for fine-grained permission management so team members can only access and modify feature flags relevant to their specific roles and responsibilities. During an audit, you'll most likely face questions about your access control systems and security measures. Here’s what auditors will want to see:
+
+-   Only authorized personnel can modify feature flags in production.
+-   All the access control changes are tracked and documented.
+-   An approval chain is in place to change critical features.
+-   Sensitive projects remain visible only to appropriate teams and/or project owners.
+
+Unleash is built with many mechanisms in place to handle all of these scenarios. Our systems enforce strict separation of duties, for example, where developers can create and test flags in development environments, while only senior engineers can promote changes to staging.
+
+-   We leave audit trails, automatically [logging every permission change](#audit-manual-and-automated-events-in-unleash) with detailed user information and timestamps.
+-   You can set up [approval guardrails](#use-a-change-management-workflow-for-auditing) for feature flag updates.
+-   [Project isolation](/reference/project-collaboration-mode) ensures sensitive projects remain hidden from unauthorized users, while teams can only access projects relevant to their work, maintaining clear boundaries between different business units' feature flags.
+
+Let’s look at how Unleash gives you complete control over user roles and permissions. At a high level, there are multiple [predefined roles](/reference/rbac#predefined-roles) in Unleash for you to get started with. The diagram below breaks down the root roles and their respective permissions. Think of them as very specific responsibilities only certain users have in Unleash that are defined by default.
+
+:::note
+
+One of the key responsibilities of the _admin role_ is assigning users as project owners and members.
+_Editor roles_ can be designated as project owners to manage projects.
```

Comment:
This is not entirely true. You can be a Viewer in the system and still be a project owner or have more permissions inside a system. Usually, following the least privilege principle, the ideal would be to make all users "Viewers" and only give specific permissions. 

In terms of security recommendations we should advice to use Viewer roles or a custom root role with minimal permissions